webots_ros2_driver/src/plugins/static/Ros2RangeFinder.cpp

```diff
@@ -45,22 +48,46 @@ namespace webots_ros2_driver
     mCameraInfoPublisher = mNode->create_publisher<sensor_msgs::msg::CameraInfo>(mTopicName + "/camera_info", cameraInfoQos);
     mCameraInfoMessage.header.stamp = mNode->get_clock()->now();
     mCameraInfoMessage.header.frame_id = mFrameName;
-    mCameraInfoMessage.height = mRangeFinder->getHeight();
-    mCameraInfoMessage.width = mRangeFinder->getWidth();
+    mCameraInfoMessage.height = height;
+    mCameraInfoMessage.width = width;
     mCameraInfoMessage.distortion_model = "plumb_bob";
-    const double focalLength = 570.34;
+    const double focalLengthx = 0.5 * width * (1 / tan(0.5 * mRangeFinder->getFov()));
+    const double focalLengthy = 0.5 * height * (1 / tan(0.5 * mRangeFinder->getFov()));
```

Comment:
I would rename these variables:
```suggestion
    const double focalLengthX = 0.5 * width * (1 / tan(0.5 * mRangeFinder->getFov()));
    const double focalLengthY = 0.5 * height * (1 / tan(0.5 * mRangeFinder->getFov()));
```